frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -167,11 +169,11 @@ export type DashboardTabId = number;
 export type DashboardTab = {
   id: DashboardTabId;
   dashboard_id: DashboardId;
-  entity_id: string;
+  entity_id?: BaseEntityId;
   name: string;
   position?: number;
-  created_at: string;
-  updated_at: string;
+  created_at?: string;
+  updated_at?: string;
```

Comment:
why do we need to change the type here?

Reply:
They're not populated anywhere in the frontend - to create a new tab, we use `getDefaultTab` which initializes the three properties there to empty strings, but they're not manipulated and nothing is done to them. We should just leave them as things that will be populated by the server once we save the dashboard tabs